src/profile-logic/profile-data.js

```diff
@@ -355,32 +355,6 @@ export type TimingsForPath = {|
   rootTime: Milliseconds, // time for all the samples in the current tree
 |};
 
-/**
- * This function Returns the JS implementation information for a specific stack.
- */
-export function getJsImplementationForStack(
```

Comment:
This was moved and changed a bit inside `getTimingsForCallNodeIndex` so that it can access all local variables directly.